### PR TITLE
Token transfer custom gateway

### PR DESCRIPTION
**Pull request by arpitB-dev**

### CODE REVIEW

**Review by ravish1729 — CHANGES_REQUESTED** [review]
See comments in respective files.

File: src/utils/config/tokenTransferData.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+export const abiURL =
+  "https://api.bscscan.com/api?module=contract&action=getabi&address=0x55d398326f99059fF775485246999027B3197955&apikey=H2JKE5NV61GF5A1XZ7Z8X79VKGDJ2A1KNW";
```
[the remark this answers is not in the document]
Comment (arpitB-dev) [reply src/utils/config/tokenTransferData.js:2]: done

File: src/utils/config/tokenTransferData.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+export const abiURL =
```
Comment (ravish1729) [inline src/utils/config/tokenTransferData.js:1]: has to be removed, abi URL. Import abi file wherever needed

File: src/utils/services/transferUSDC.js [hunk 3]
```diff
@@ -1,70 +1,55 @@
 import { ethers } from "ethers";
-import { notify } from "./notification";
-
+import axios from "axios";
+import {
+  tokenCost,
+  receiverAddress,
+  contractAddress,
+  abiURL,
```
Comment (ravish1729) [inline src/utils/services/transferUSDC.js]: again abi file insted of url

**Review by ravish1729 — CHANGES_REQUESTED** [review]
Minor updates needed

File: src/pages/adminPanel/TopUp/TopUp.jsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+            <p>Topup Lighthouse Storage</p>
+        </div>
+
+        <div className="topup__desctription">
+            <p>Add Topup from our value added packages:</p>
+        </div>
+
+        <div className="topup__content">
+
+            <div className="planCard">
```
Comment (nandit123) [inline src/pages/adminPanel/TopUp/TopUp.jsx:57]: change UI for topup as discussed

File: src/utils/config/tokenTransferData.js (hunk 2 above)
Comment (nandit123) [reply src/utils/config/tokenTransferData.js:1]: good catch

**Review by nandit123 — CHANGES_REQUESTED** [review]
To correct changes pointed by ravish, rest LGTM

File: src/utils/services/transferUSDC.js (hunk 3 above)
Comment (arpitB-dev) [reply src/utils/services/transferUSDC.js]: done

**Pull request closed without merging**